src/Component/RequestBuilder/FacetSet.php

```diff
@@ -108,28 +118,31 @@ public function buildComponent($component, $request)
      *
      * @param Request    $request
      * @param FacetField $facet
+     * @param compatible $compatible false if local parameters should be used
```

Comment:
The comment has to be enhanced to explain when local params make sense, just a short sentence. And we should mention https://issues.apache.org/jira/browse/SOLR-6193 to explain why local params aren't always used.